sdk/storage/src/blob/blob/block_list.rs

```diff
@@ -1,37 +1,30 @@
 use crate::blob::blob::{BlobBlockType, BlockWithSizeList};
-use std::borrow::Borrow;
 
 #[derive(Default, Debug, Clone, PartialEq)]
-pub struct BlockList<T>
-where
-    T: Borrow<[u8]>,
-{
-    pub blocks: Vec<BlobBlockType<T>>,
+pub struct BlockList {
+    pub blocks: Vec<BlobBlockType>,
 }
 
-impl<'a> BlockList<&'a [u8]> {
-    pub fn to_owned(&self) -> BlockList<Vec<u8>> {
-        let mut bl: BlockList<Vec<u8>> = BlockList {
+impl BlockList {
+    pub fn to_owned(&self) -> Self {
```

Comment:
Why is this not just a `clone` implementation?

Reply:
You are right, it made sense before because it converted from borrowed to owned data. 
Now, with bytes, clone is enough! Good catch!